sc2/unit.py

```diff
@@ -1177,6 +1188,11 @@ def weapon_cooldown(self) -> float:
             return self._proto.weapon_cooldown
         return -1
 
+    @property
+    def weapon_ready(self) -> bool:
+        """ Checks if the weapon is ready to be fired. """
+        return self.weapon_cooldown == 0
```

Comment:
Could be optimized to `return not self.weapon_cooldown`, but I think it's fine

Reply:
I think I actually prefer comparison against zero here. That makes it explicit.

I thought there was a property like this already, but perhaps I am mistaken.